### ISSUE TITLE
Multilingual support?

### ISSUE THREAD

**Issue by kurax** [screeps/docs#105]
Is it possible to make the site multilingual so we can contribute translations rather than fork and deploy my own?

**Comment by RiQuY** [comment]
Yeah nice idea, we will love to help the devs to translate the game with the proper tools like websites like Crowdin.
